apps/blender/resources/images/entrypoints/scripts/verifier_tools/verificator.py

```diff
@@ -65,7 +66,7 @@ def prepare_params(mounted_paths, subtask_border, scene_file_path, resolution, s
     return crops, params
 
 
-def make_verdict( subtask_file_paths, crops, results ):
+def make_verdict(subtask_file_paths, crops, results, raw_verification):
```

Comment:
hm... maybe `use_raw_verification`?
```suggestion
def make_verdict(subtask_file_paths, crops, results, use_raw_verification):
```

Reply:
Done